### PR TITLE
fixed #36: added radio buttons

### LINKED ISSUE #36: Implement radio buttons
We need someone to implement radio buttons.

Please feel free to ask for help, if you want to work on this.

### PR DESCRIPTION
**Pull request by superzazu**

### CODE REVIEW

File: examples/radiobuttons.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+"""
+ Shows a simple combobox.
```
Comment (nsgonultas) [inline examples/radiobuttons.py]: A little typo: *combobox*

File: tests/test_radiobuttons.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class RadioButtonsTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.radio_buttons = RadioButtons()
+
+    def test_set_selected(self):
+        """Tests the setSelected method of the combobox."""
```
Comment (nsgonultas) [inline tests/test_radiobuttons.py]: Again, not *combobox* :slightly_smiling_face:

**Comment by superzazu** [comment]
Whoops, thanks for correcting me. Pushed another commit to fix those typos :-)

**Comment by nsgonultas** [comment]
Thanks for PR :+1:

**Pull request merged**